minitests/bram-sdp/runme.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -ex
+${XRAY_VIVADO} -mode batch -source ../runme.tcl
+${XRAY_BITREAD} -F $XRAY_ROI_FRAMES -o design.bits -z -y design.bit
+test -z "$(fgrep CRITICAL vivado.log)"
+${XRAY_SEGPRINT} -z -D design.bits > design.txt
+${XRAY_BIT2FASM} design.bit > design.fasm
```

Comment:
For minitests like this, it is always a good idea to supply `--verbose` to ensure that unknown features are reported (if any).

Reply:
Sure, done.